src/authMiddleware/authMiddleware.js

```diff
@@ -10,11 +9,10 @@ export function authMiddleware(request) {
   let isAuthenticated = false;
   const nextUrl = trimTrailingSlash(request.nextUrl.href);
   const logoutUrl = trimTrailingSlash(config.postLogoutRedirectURL);
-  const kinde_token = request.cookies.get('kinde_token');
+  const kinde_token = request.cookies.get('access_token');
```

Comment:
@peterphanouvong is this right?

Reply:
@DaveOrDead yea, using TS SDK sets `access_token` in cookies instead of `kinde_token`

Reply:
Hmm I see. Seems this would create breaking changes for anyone accessing the cookie directly.

Could we instead update the TS SDK to allow a cookie name to be passed? That way we can pass `kinde_token` from the NextJS SDK which keeps it working as it was for existing users?